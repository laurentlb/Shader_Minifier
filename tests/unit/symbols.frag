#   extension GL_EXT_gpu_shader4 : enable
#  define TEST
# ifndef A
# endif

vec3 color; // TLDecl

struct s { // TypeDecl
  vec4 n, f;
  bool ok;
};

float sdf(vec3 p) { // Function
  color = vec3(1.);
  return length(p) - 1.;
}

void main() {
    int ijk = 32;
    ijk += ijk;
}