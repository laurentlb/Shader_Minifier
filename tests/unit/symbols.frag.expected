#extension GL_EXT_gpu_shader4:enable

#define TEST

#ifndef A

#endif

vec3 color;struct s{vec4 n,f;bool ok;};
float sdf(vec3 p)
{
  color=vec3(1);
  return length(p)-1.;
}
void main()
{
  int ijk=32;
  ijk+=ijk;
}